.github/CODEOWNERS

```diff
@@ -1,6 +1,2 @@
 # These owners will be the default owners for everything in the repo.
-*       @markusdregi
-*       @lars-petter-hauge
-*       @andreabrambilla
-*       @pgdr
-*       @jokva
+*       @markusdregi @lars-petter-hauge @andreabrambilla @pgdr @jokva
```

Comment:
Shall @xjules  be there also?

Reply:
Sure ;)

Reply:
I'll merge this one, add you in a second PR to see that it actually works